src/super_gradients/training/transforms/transforms.py

```diff
@@ -690,74 +703,84 @@ def __init__(
         self.maybe_flip = DetectionHorizontalFlip(prob=flip_prob)
 
     def close(self):
-        self.additional_samples_count = 0
         self.enable_mixup = False
 
-    def apply_to_sample(self, sample: DetectionSample) -> DetectionSample:
-        if self.enable_mixup and random.random() < self.prob:
-            (cp_sample,) = sample.additional_samples
-            target_dim = self.input_dim if self.input_dim is not None else sample.image.shape[:2]
-
-            cp_sample = self.maybe_flip.apply_to_sample(cp_sample)
-
-            jit_factor = random.uniform(*self.mixup_scale)
+    @property
+    def additional_samples_count(self) -> int:
```

Comment:
May I suggest moving this propery after `apply_to_sample` method? Purely to improve diff visualization (GH struggles to understand this method is present and shows odd diff)

Reply:
[e3aeabc](https://github.com/Deci-AI/super-gradients/pull/1897/commits/e3aeabc3dcce9cbc0812e8a44f37569017601d57)